### ISSUE TITLE
move tests into own files

### ISSUE THREAD

**Issue by patkan** [fau-fablab/FabLabKasse#90]
Hi,
I would like to move all tests in separate files in their respective directories. The way we are doing it currently seems to be problematic since we try to execute all files but only some include tests.
Moreover this would be cleaner: we could separate the includes and code for tests and real logic.
If no one opposes until tonight, I will start separating the tests.

**Comment by patkan** [comment]
@fau-fablab/kasse please have a look at this issue.

**Comment by sedrubal** [comment]
Even if my contribution to kasse was not that much, I think this would be a good idea...

**Comment by mgmax** [comment]
no, we don't execute all files. all files must be importable, and all unittest.testcase subclasses are run.

**Comment by patkan** [comment]
Ok, you are right, my bad.
We search every source file, while typical test systems assume that tests reside in certain files (e.g. test_*.py)
The background for this that I try to get nearer to the de-facto-standards.
Currently, Travis is failing while building the unit test. I don't think this will fix the problem with Travis, but maybe it makes it easier to find errors.

**Comment by patkan** [comment]
I drop this task for now because I do not have the time at the moment.
